OWNERS

```diff
@@ -1,11 +1,13 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
+  - accorvin
   - anishasthana
+  - gmfrasca
   - HumairAK
-  - tumido
+  - maulikjs
+  - rimolive
   - sesheta
+  - tumido
 reviewers:
-  - durandom
```

Comment:
@anishasthana I think you want us in the reviewers list as well. As far as I understand on how this works, `prow`  requests review from "somebody" from the `reviewers` list. So if we don't want all the reviews to be assigned to @harshad16 , we should ourselves in here as well.

Reply:
Ah.. Got it.